python/ql/test/experimental/dataflow/regression/test.py

```diff
@@ -211,19 +211,19 @@ def flow_in_generator():
 
 def flow_from_generator():
     for x in flow_in_generator():
-        SINK(x)
+        SINK(x)  # Flow not found
 
 def const_eq_clears_taint():
     tainted = SOURCE
     if tainted == "safe":
-        SINK(tainted) # safe
+        SINK(tainted) # safe  # FP
     SINK(tainted) # unsafe
 
 def const_eq_clears_taint2():
     tainted = SOURCE
     if tainted != "safe":
         return
-    SINK(tainted) # safe
+    SINK(tainted) # safe  # FP
```

Comment:
I'm not sure whether these tests makes sense for dataflow. I think they are more appropriate for taint tracking.

Reply:
True, perhaps we just remove them.